packages/blockchain/test/index.ts

```diff
@@ -585,9 +593,13 @@ test('blockchain test', t => {
         if (err) {
           return st.error(err)
         }
-        st.equals(head.hash().toString('hex'), genesis.hash().toString('hex'), 'should get head')
-        st.equals(blockchain._heads['head0'].toString('hex'), 'abcd', 'should get state root heads')
-        st.end()
+        if (genesis) {
+          st.equals(head.hash().toString('hex'), genesis.hash().toString('hex'), 'should get head')
+          st.equals(blockchain._heads['head0'].toString('hex'), 'abcd', 'should get state root heads')
+          st.end()
+        } else {
+          st.fail()
+        }
```

Comment:
Similar question here on these new conditional block defined yes/no switches: does this just make things cleaner or does it fix some newly triggered errors?